addons/vrm/import_vrm.gd

```diff
@@ -23,13 +23,10 @@ func _import_animation(path: String, flags: int, options: Dictionary, bake_fps:
 
 
 func _import_scene(path: String, flags: int, options: Dictionary, bake_fps: int) -> Node:
-	var vrm_loader = load("res://addons/vrm/vrm_loader.gd").new()
+	var gltf : GLTFDocument = GLTFDocument.new()
+	var extension : GLTFDocumentExtension = load("res://addons/vrm/vrm_extension.gd").new()
+	gltf.extensions.push_front(extension)	
+	var state : GLTFState = GLTFState.new()
+	var root_node = gltf.append_from_file(path, state, flags, bake_fps)
```

Comment:
This returns an error code now according to Godot master, so the error code should either be handled or not stored in a variable named `root_node`

Reply:
Changed when able